### DISCUSSION TITLE
how to hex in messages.yml

### DISCUSSION THREAD

**Discussion by PolsatGraniePL** [games647/FastLogin#816] [Q&A] [answered]
.

**Comment by PolsatGraniePL** [comment]
![image](https://user-images.githubusercontent.com/88681446/173569495-1e9cf23e-268e-4281-896c-ca857c61431b.png)
are there any other ways?

**Comment by TuxCoding** [comment]
It isn't possible in FastLogin at the moment, because it needs to parsed by it.

**Reply by PolsatGraniePL** [reply]
;(

and when will it be possible?

**Comment by TuxCoding** [comment, marked as answer]
I looked into it and it looks like it's possible. You could use `&xHEX`. It was implemented by BungeeCord and is used in Spigot too.

**Comment by PolsatGraniePL** [comment]
when I use it it always pops up black
![image](https://user-images.githubusercontent.com/88681446/174657500-625012d0-fcd9-4ba2-aed6-629da6fd96f4.png)
![image](https://user-images.githubusercontent.com/88681446/174657518-ff3e3ef6-53fc-4d5f-9564-d6c61468364b.png)

am i doing something wrong?

**Reply by TuxCoding** [reply]
Yes, you are missing the `x` character and the one color shouldn't be split up. So you need `&x00002a`.

**Reply by PolsatGraniePL** [reply]
![image](https://user-images.githubusercontent.com/88681446/174749822-604ca032-4fa9-4660-93aa-aed27c090fff.png)
`'&x00002a&lABC'`
![image](https://user-images.githubusercontent.com/88681446/174749982-21f9500e-fed6-4eee-bca3-122448f5885f.png)

It didn't work so I took something that supposedly "works" because it gave a black color.
and what you wrote, it doesn't work at all. I have the latest version of fastlogin

**Reply by TuxCoding** [reply]
Sorry, but the BungeeCord component is not very documented. Looking at the code, it parses 12 characters after the x. This could indicate foreground and background color. You could try `§x00002a00002b§lText` or the unparsed one with `&` instead of `§`. I tested that with BungeeCord and it works, but it might requires some work for Spigot.
